crates/sncast/tests/data/scripts/script_manual_tests/Scarb.toml

```diff
@@ -0,0 +1,24 @@
+[package]
```

Comment:
not sure we should put `manual tests` in the repo at this stage. It would be better to just make it a regular test that runs on devnet, that you happen to be able to easily run on real network (as it is with map_script at the moment). 

I'm just not sure anyone will remember that something like this exists somewhere in our tests files, as it is not mentioned nor used aywhere.